examples/websocket-retry-client/websocket_retry_client.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how a WebSocket client reconnects to the same backend when the client looses the connection.
```

Comment:
BBE on how transient failures can be handle using the WebSocket client. When it comes to WebSocket, the key attribute is maintaining a live connection. This BBE shows how WebSocket client can be made resilient with its reconnect feature.

Reply:
Fixed with ce5d648